messaging-experiments.yaml

```diff
@@ -1,22 +1,166 @@
 - id: AW_PULL_FACTOR_PRIVACY_1
-  filter_expression: localeLanguageCode == "en" && browserSettings.update.channel ==
-    "release" && firefoxVersion >= 75 && [userId, "aw_pull_factor_privacy_1"]|bucketSample(0,
-    2000, 10000)
+  enabled: true
+  filter_expression: localeLanguageCode == "en" &&
+    firefoxVersion >= 76 && [userId, "aw_pull_factor_privacy_1"]|bucketSample(0,
+    2000, 10000) &&
```

Comment:
&& [userId, \"aw_pull_factor_privacy_1\"]|bucketSample(0, 2000, 10000) part in filter_expression is not evaluating to true when trying in local build, is that expected?

Reply:
`[userId, "aw_pull_factor_privacy_1"]|bucketSample(0, 2000, 10000)` should randomly return `true` or `false` depending on your specific `userId` (20% of users should get `true`)